forage-android/src/main/java/com/joinforage/forage/android/model/EncryptionKey.kt

```diff
@@ -3,14 +3,19 @@ package com.joinforage.forage.android.model
 import org.json.JSONObject
 
 internal data class EncryptionKey(
```

Comment:
nit: we should consider renaming `EncryptionKey` -> `EncryptionKeys`^

we refer to the var of this type as `encryptionKeys` across this PR